runauto_nag_daily.sh

```diff
@@ -4,5 +4,114 @@ set -e
 export PYTHONPATH=.
```

Comment:
Double-check you didn't miss anything when moving from the now deleted runauto_nag_... files

Reply:
Only `regression_but_type_enhancement_task` seems to be missing, in the hourly runner script.